### ISSUE TITLE
[libretro] General issues with layers when toggling between fullscreen/windowed

### ISSUE THREAD

**Issue by ner00** [FCare/Kronos#892]
This is not something common to happen, and I'm not sure if it is already known, but while doing some testing I alternated between fullscreen and windowed mode and the picture went black. By pressing buttons randomly some game event refreshed the screen (or layers) and the image seems to have been restored, but some bugs remained.

The first issue is really everything going completely black in the worst case scenario. The best case scenario is that some layers will be missing or inaccurate, here's one example below - left is before toggling, right is after toggling:

![](https://i.imgur.com/tCce2sv.png)

**Comment by barbudreadmon** [comment]
Yes, the issue is known, i guess it's not a bad idea to open a br about it though. The issue is about how per design libretro destroy gl context when toggling fullscreen/windowed, so every gl bits need to be properly deinitialized before being reinitialized again, actually it was a lot worse until recently (core would generally crash).

**Comment by BenjaminSiskoo** [comment]
I close this one if @ner00 have the same problem, I'll open it again

**Comment by barbudreadmon** [comment]
I don't think this br should be closed
